src/mudlet.cpp

```diff
@@ -559,8 +559,10 @@ void mudlet::slot_module_clicked(QTableWidgetItem* pItem){
     qDebug()<<itemPath->text();
     if (!entry || !checkStatus || !itemPriority || !pH->mInstalledModules.contains(entry->text())){
         moduleHelpButton->setDisabled(true);
-        checkStatus->setCheckState(Qt::Unchecked);
-        checkStatus->setFlags(Qt::NoItemFlags);
+        if (checkStatus) {
```

Comment:
We have already validated `checkStatus` is valid in line 561 have we not - oh no - that not being valid is one way to get here :open_mouth:.  This ___coverity___ thing __is__ _good stuff_ isn't it! :+1: